src/components/Popover/src/PopoverContent.vue

```diff
@@ -33,9 +41,19 @@ export default {
 
 	computed: {
 		computedStyles() {
+			// PopoverContent can be rendered outside the current Mtheme wrapper
+			// depending on placement of MPopoverLayer so we need to make sure
+			// it receives the correct Mtheme color variables
+			const themeColorVars = {};
+			Object.entries(this.theme.colors).forEach(([color, hex]) => {
```

Comment:
It's unfortunate that we have to reprocess the values again here into CSS, but this is the best solution for now.

Ideally we can simplify this later by generating the CSS at the top of the theme for the default colors and profiles -- at that point we could remove this and optimize with setting a class on the popover.